OKR/PROJECT.md

```diff
@@ -2,8 +2,27 @@
 
 We frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
 
-## 2018 Q4
+## 2019 Q1
```

Comment:
@momack2 @meiqimichelle @parkan did a first pass at grabbing the ideas on https://github.com/ipfs/team-mgmt/pull/793 and converted them to the list below. Please review.